### PR TITLE
Remove queries for internal transactions

### PR DESCRIPTION
**Pull request by paullinator**
Internal transactions are by definition between contracts and don't affect the end balance of an EVM account. These queries were overwriting regular transactions with the same txid causing incorrect transaction amounts.

### CODE REVIEW

File: src/ethereum/ethNetwork.js [hunk 1]
```diff
@@ -243,6 +242,12 @@ export class EthereumNetwork {
     } else {
       throw new Error('Invalid transaction result format')
     }
+    if (
```
Comment (peachbits) [inline src/ethereum/ethNetwork.js:245]: Remove this debugging log

**Review by peachbits — CHANGES_REQUESTED** [review]

File: src/ethereum/ethNetwork.js (hunk 1 above)
Comment (paullinator) [reply src/ethereum/ethNetwork.js:245]: good catch

File: src/ethereum/ethNetwork.js [hunk 2]
```diff
@@ -41,7 +41,6 @@ import {
   type FetchGetAmberdataApiResponse,
   type RpcResultString,
   asAlethioAccountsTokenTransfer,
-  asAmberdataAccountsFuncs,
```
Comment (peachbits) [inline src/ethereum/ethNetwork.js:44]: Duh I didn't catch these wouldn't be used anymore. Can you remove them the ethTypes too?

**Review by peachbits — CHANGES_REQUESTED** [review]

**Review by peachbits — APPROVED** [review]

**Pull request merged**